### PR TITLE
AWS driver updates and to carry user's pr

### PR DESCRIPTION
**Pull request by moxiegirl**
Carries and closes  #1352
@kacole2 please make sure your change was carried correctly
@dmp42 Carried per PR

Signed-off-by: Mary Anthony <mary@docker.com>

### CODE REVIEW

File: docs/install-machine.md [hunk 1]
```diff
@@ -31,39 +31,35 @@ target="_blank">the Docker binary</a>.
...
+        mv docker-machine* /usr/local/bin
+
+    Windows (using Git Bash):
+
+        $ curl -L https://github.com/docker/machine/releases/download/v0.5.0/docker-machine_windows-amd64.zip >machine.zip && \
+        unzip machine.zip && \
+        rm machine.zip && \
+        mv docker-machine* /usr/local/bin
+
+3. Make sure the file is executable.
```
Comment (dmp42) [inline docs/install-machine.md]: @nathanleclaire is there any reason why this would still be necessary?

**Comment by dmp42** [comment]
Thanks @moxiegirl 
Open question about the "chmod this" part to resolve, then should be good to go.

File: docs/install-machine.md (hunk 1 above)
Comment (nathanleclaire) [inline docs/install-machine.md]: It doesn't seem to be needed anymore and should be removed.

**Comment by nathanleclaire** [comment]
Like @dmp42 said, once the `chmod` instruction is removed I'm good to go on this.

**Comment by moxiegirl** [comment]
Done.  Thanks guys.

File: docs/install-machine.md [hunk 2]
```diff
@@ -31,39 +31,31 @@ target="_blank">the Docker binary</a>.
...
+        mv docker-machine* /usr/local/bin
+
+    Windows (using Git Bash):
+
+        $ curl -L https://github.com/docker/machine/releases/download/v0.5.0/docker-machine_windows-amd64.zip >machine.zip && \
+        unzip machine.zip && \
+        rm machine.zip && \
+        mv docker-machine* /usr/local/bin
+
+4. Check the installation by displaying the Machine version:
```
Comment (dmp42) [inline docs/install-machine.md]: Wouldn't that be `3.`?

**Comment by nathanleclaire** [comment]
LGTM

**Pull request merged**